src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpProxyCacheableRequestGroup.java

```diff
@@ -16,23 +16,42 @@
 package org.reaktivity.nukleus.http_cache.internal.stream;
 
 import static org.reaktivity.nukleus.http_cache.internal.stream.Signals.CACHE_ENTRY_ABORTED_SIGNAL;
+import static org.reaktivity.nukleus.http_cache.internal.stream.Signals.CACHE_ENTRY_NOT_MODIFIED_SIGNAL;
 import static org.reaktivity.nukleus.http_cache.internal.stream.Signals.CACHE_ENTRY_UPDATED_SIGNAL;
-import static org.reaktivity.nukleus.http_cache.internal.stream.Signals.INITIATE_REQUEST_SIGNAL;
+import static org.reaktivity.nukleus.http_cache.internal.stream.Signals.REQUEST_GROUP_LEADER_UPDATED_SIGNAL;
 
+import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 import java.util.function.Consumer;
 
+import org.agrona.DirectBuffer;
 import org.agrona.collections.Long2LongHashMap;
+import org.agrona.collections.LongHashSet;
+import org.agrona.collections.MutableInteger;
+import org.agrona.collections.MutableLong;
+import org.reaktivity.nukleus.function.MessageConsumer;
 import org.reaktivity.nukleus.http_cache.internal.stream.util.Writer;
+import org.reaktivity.nukleus.http_cache.internal.types.stream.BeginFW;
 
 final class HttpProxyCacheableRequestGroup
 {
-    private final Long2LongHashMap routeIdsByReplyId;
-    private final int requestHash;
+    private final Map<String, Long2LongHashMap> requestsQueue;
+    private final LongHashSet responsesInFlight;
     private final Writer writer;
     private final HttpCacheProxyFactory factory;
     private final Consumer<Integer> cleaner;
+    private final MutableLong activeRouteId;
+    private final MutableLong activeReplyId;
+    private final int requestHash;
+
+    private MessageConsumer connect;
+    private String etag;
     private String recentAuthorizationToken;
+    private long acceptRouteId;
+    private long acceptReplyId;
+    private long routeId;
+    private long replyId;
```

Comment:
What is the distinction between `acceptRouteId` vs `routeId` and `acceptReplyId` vs `replyId`?

Reply:
- `acceptRouteId`  is redundant I removed it.  
- `replyId` is connectReplyId. I have changed it.
- `routeId ` is `connectRouteId. I have changed it.